ml-agents/mlagents/trainers/tests/test_simple_rl.py

```diff
@@ -263,9 +263,12 @@ def _check_environment_trains(
         # Begin training
         tc.start_learning(env_manager)
         print(tc._get_measure_vals())
-        for mean_reward in tc._get_measure_vals().values():
```

Comment:
Seems this change should be part of another PR

Reply:
Moved to separate PR #3511